spk/emby/src/dsm-control.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/sh
+
+# Package
+PACKAGE="emby"
+DNAME="Emby"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+PATH="${INSTALL_DIR}/bin:${PATH}"
+USER="${PACKAGE}"
+USER_HOME="$(eval echo ~$USER)"
```

Comment:
Maybe just set it to the same path you use in the installer (https://github.com/SynoCommunity/spksrc/pull/1673/files#diff-af6eb7962b714d2e23e6d6895ba91cd0R38). I see you also define `DATA_DIR`, it's the same thing really.